float64[100] local_x
float64[100] local_y
float64[100] local_theta
